### ISSUE TITLE
Proposition de modification du style des articles

### ISSUE THREAD

**Issue by SL-prog** [Sivigik/Sivigik#2]
![sivigik_article](https://cloud.githubusercontent.com/assets/6438281/4853658/26ce862e-6088-11e4-9b55-3b24704a74df.png)

**Comment by Klafyvel** [comment]
Tu propose le titre de l'article en incrustation sur l'image c'est ça ?

**Comment by SL-prog** [comment]
Oui, afin de pouvoir adapter plus facilement les images sur support mobile. Après, faut-il mettre un titre de la même police pour chaque article ou faut-il laisser le choix de la police au rédacteur ? Je pense que le premier serait plus beau mais que le deuxieme serait plus dur à mettre en place. Il faut donc choisir une police qui passe bien niveau style et couleurs sur tous les articles.

**Comment by Klafyvel** [comment]
C'est pas ce que je voulais dire, je pensais mettre l'image sans titre pour le slider et après grâce à css mettre le titre par dessus.

**Comment by SL-prog** [comment]
Ben oui c'est ce que je dis.

**Comment by Klafyvel** [comment]
ok on se comprend ;)
